crates/next-api/src/project.rs

```diff
@@ -754,6 +754,18 @@ impl Project {
         }
     }
 
+    #[turbo_tasks::function]
+    pub(super) fn chunking_context(
```

Comment:
Maybe call in `runtime_chunking_context` to not confuse it with the client-side chunking context